docs/content/toc.textile

```diff
@@ -16,7 +16,7 @@ h2. Data Ingestion
 * "Schema Design":../ingestion/schema-design.html
 * "Schema Changes":../ingestion/schema-changes.html
 * "Realtime Ingestion":../ingestion/realtime-ingestion.html
-* "Batch Ingestion":../ingestion/batch-ingestion.html
+* "Batch ingestion or Re-indexing or Delta ingestion":../ingestion/batch-ingestion.html
```

Comment:
I think Re-indexing should be a completely different section in the ingestion docs with these explicit steps listed out